tests/pytests/unit/transport/test_zeromq.py

```diff
@@ -14,8 +22,235 @@
 import salt.utils.platform
 import salt.utils.process
 import salt.utils.stringutils
+from salt.master import SMaster
 from tests.support.mock import MagicMock, patch
 
+try:
+    from M2Crypto import RSA
+
+    HAS_M2 = True
+except ImportError:
+    HAS_M2 = False
+    try:
+        from Cryptodome.Cipher import PKCS1_OAEP
+    except ImportError:
+        from Crypto.Cipher import PKCS1_OAEP  # nosec
+
+log = logging.getLogger(__name__)
+
+MASTER_PRIV_KEY = """
```

Comment:
Should these be fixtures too instead of globals?

Reply:
Considering they are just static strings with no setup and teardown. I don't think it matters. Maybe a purist would disagree.

Reply:
Comment based on previous reviews where we seem to want to avoid using global variables in tests.  Not a blocker, just something I noticed when reviewing.